examples/neuralNetwork-sequence-hand-gestures/index.html

```diff
@@ -0,0 +1,42 @@
+<!--
```

Comment:
Nitpick: `hand-gestures` is plural while `mouse-gesture` is singular. (Maybe singular throughout?)